test/test_interfacevalues.jl

```diff
@@ -6,8 +6,8 @@
 
         @test getnbasefunctions(iv) == n_basefuncs
 
-        for face_a in topology.face_skeleton
-            neighbors = dim > 1 ? topology.face_neighbor[face_a[1], face_a[2]] : topology.vertex_neighbor[face_a[1], face_a[2]]
+        for face_a in faceskeleton(topology, grid)
+            neighbors = dim > 1 ? topology.face_face_neighbor[face_a[1], face_a[2]] : topology.vertex_vertex_neighbor[face_a[1], face_a[2]]
```

Comment:
Well, this one is still inconsistent :/ @koehlerson @fredrikekre since we have this inconsistency in the codebase I might have to ask whether in 1D the vertex is a face or not?

Also, interface.
```suggestion
            neighbors = dim > 1 ? getneighborhood(...) : ...
```

Reply:
I don't know and honestly I don't think that there is any truly good solution to it for d=1,d=2 and d=3. I do think there are nice definitions for d=2, d=3 but as soon as you include 1D or vice verca it becomes quite fast artificially named or defined, but these are just my two cents . So I personally don't mind if its vertex or face neighborhood as long as its stated/defined somewhere